### PR TITLE
Add JS to content-aside and functionality for timeline component on mobile

### PR DESCRIPTION
**Pull request by Cosmin-Daniel**

### CODE REVIEW

**Comment by nlisgo** [comment]
I will postpone reviewing until after the following is merged: https://github.com/elifesciences/pattern-library/pull/1210

Also, could do with @chris-huggins taking a look.

**Review by chris-huggins — COMMENTED** [review]
Please see comments on https://github.com/elifesciences/issues/issues/7935

File: assets/sass/patterns/organisms/content-aside.scss
Comment (discodavey) [inline assets/sass/patterns/organisms/content-aside.scss:31]: Can we not have this CSS in the contextual data css file rather than having it in two places

**Review by discodavey — COMMENTED** [review]
When you reduce browser size the toggle does not kick in

**Review by discodavey — COMMENTED** [review]
When on breakpoints 730-900 the toggle only hides one of the items where as below 730 it hides all. 

I think this is making things too complex. Either this toggle should show all or hide all.

May need to speak to @chris-huggins on this as this maybe something he may have wanted.

File: assets/sass/patterns/organisms/content-aside.scss
Comment (viktoria-mahmud) [reply assets/sass/patterns/organisms/content-aside.scss:31]: I'm not sure I understand correctly - do we need to move just the "contextual-data__list" css to the existing file "contextual-data.scss" ?

**Comment by Cosmin-Daniel** [comment]
> When on breakpoints 730-900 the toggle only hides one of the items where as below 730 it hides all.
> 
> I think this is making things too complex. Either this toggle should show all or hide all.
> 
> May need to speak to @chris-huggins on this as this maybe something he may have wanted.

Yes, these were the requests. You can check the design [here](https://projects.invisionapp.com/d/main#/console/22177282/470910740/preview).

File: assets/sass/patterns/organisms/content-aside.scss
Comment (viktoria-mahmud) [reply assets/sass/patterns/organisms/content-aside.scss]: Done, contextual-data css is removed from the content-aside.scss file to the contextual-data.scss file

**Review by discodavey — COMMENTED** [review]
Toggle is broken below 320. I know we support from 320 up but can we make the toggle work show/hide below 320 as well.

**Comment by discodavey** [comment]
I'm happy with the changes.

**Review by discodavey — APPROVED** [review]

**Comment by nlisgo** [comment]
If you start on the desktop view and resize the browser window to the narrow viewport the expectation is that the toggle would be there and the default would show one item in timeline for narrow and three for medium. Actual behaviour is that the toggle doesn't appear in that scenario. The toggle only appears if browser window is not wide on page load.

**Comment by chris-huggins** [comment]
@nlisgo Yes they work as expected and look great, thank you.

**Comment by nlisgo** [comment]
@Cosmin-Daniel can you verify that you are happy with the current state of this and approve if so.

**Comment by Cosmin-Daniel** [comment]
> @Cosmin-Daniel can you verify that you are happy with the current state of this and approve if so.

Everything seems to work ok. There is only the situation when the last DT is on more than one row, but if you've talked with Chris and he agreed for this behaviour then I'm ok with it.

**Comment by nlisgo** [comment]
@chris-huggins just added one more commit to remove the truncate with ellipsis on the first dt on smaller viewport sizes. I understand from our conversation yesterday that that needed to go.

**Pull request merged**
